index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
```

Comment:
There are other signals that signal termination, not only `SIGKILL` (the default value of `process.kill()`). You can get them from `signalExit.signals()` (`signal-exit` is already a dependency).

See https://github.com/tapjs/signal-exit/blob/master/signals.js

Reply:
Mh... I wonder if we should also consider those. The thing is that the default implementation for kill [defaults](https://github.com/nodejs/node/blob/f5e2ad784ff8d51db275a7268660fc72cf58289d/lib/internal/child_process.js#L438) to SIGTERM. So to keep the API clean the thought was that people that don't care about the _killing_ usage gain benefit while people that perform weird signal passing don't kill their process by accident.
Having only a condition on SIGTERM ensures that the purpose of calling `kill` was to destroy the existence of the process in a nice way.

Reply:
I see what you mean.

On one hand, if the signal was coming from the user, we should definitely take into account those extra signals. For example, `SIGINT` is done with `CTRL-C`, and also `SIGQUIT` is done with `CTRL-\` on Windows `cmd.exe`.

But here it's programmatic. Except for the case where signals would be forwarded from parent to child (which should be uncommon), your points are valid.

Also only `SIGTERM`, `SIGKILL` and `SIGTERM` [work on Windows](https://github.com/ehmicky/cross-platform-node-guide/blob/master/docs/networking_ipc/signals.md#cross-platform-signals).

So I agree with you, we should stick to `SIGTERM` only. @sindresorhus, what do you think?